Source/WebCore/page/scrolling/ScrollingTree.cpp

```diff
@@ -56,6 +57,14 @@ struct CommitTreeState {
     HashSet<ScrollingNodeID> synchronousScrollingNodes;
     // orphanNodes keeps child nodes alive while we rebuild child lists.
     OrphanScrollingNodeMap orphanNodes;
+    // Hosted subtrees needing attaching to scrolling tree after main commit has finished
+    Vector<std::pair<LayerHostingContextIdentifier, Vector<std::unique_ptr<ScrollingStateTree>>>> pendingSubtreesNeedingCommit;
```

Comment:
Do we need the possibility of having null ScrollingStateTrees, or can this be a UniqueRef<ScrollingStateTree>?